### PR TITLE
Prepare a 0.5.0 release.

### PR DESCRIPTION
**Pull request by ltratt**
Please check `CHANGES.md` carefully to make sure that I've captured everything correctly!

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,98 @@
...
+    ```
+
+  will produce a warning that the `mut` part of `&mut` is no longer needed.
+
+
+## Major changes
+
+* Add support for zero-copying user input when parsing. A special lifetime
+  `'input` is now available in action code and allows users to extract parts
+  of the input with out calling `to_owned()` (or equivalent). For example:
```
Comment (ptersilie) [inline CHANGES.md]: `with out` -> `without`?

**Comment by ptersilie** [comment]
One possible typo, otherwise looks good.

File: CHANGES.md (hunk 1 above)
Comment (ltratt) [reply CHANGES.md]: Fixed in https://github.com/softdevteam/grmtools/pull/139/commits/77597ba7eea698d8ec0099cfb4a3b312d55fa160.

**Comment by ptersilie** [comment]
Please squash.

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**